### DISCUSSION TITLE
32 bit Version

### DISCUSSION THREAD

**Discussion by Zft0** [nroduit/Weasis#308] [Q&A]
Dear Nicolas,
There are some very interesting features in version 4 that are not supported in version 3.6.
Is it possible that we have MSI package for 32 bit architecture as well?

**Comment by nroduit** [comment]
No, because in OpenJDK some tools do not support 32-bit anymore and the memory management in Weasis sometimes leads to crashes.

The end of life [has started for Windows 32-bit](https://pureinfotech.com/windows-10-32-bit-end-life/) and the number of running systems becomes insignificant.

**Comment by nroduit** [comment]
- Support of [Windows 32-bit](https://community.f-secure.com/total-en/kb/articles/29675-support-for-windows-10-32-bit-ending-on-september-30-2024)
- [ OpenJDK support](https://openjdk.org/jeps/479)
